README.md

```diff
@@ -2,15 +2,15 @@
 
 This example demonstrates how to use the Mbed OS file system.
 
-More information about the Mbed OS file system and other related pieces of the Mbed OS storage stack can be found [here](https://os.mbed.com/docs/v5.6/reference/storage.html).
+You can find more information about the Mbed OS file system and other related pieces of the Mbed OS storage stack [in the storage overview](https://os.mbed.com/docs/v5.6/reference/storage.html).
 
 ## Hardware requirements
 
 This example uses a block device as storage. This can be either an external
-block device (one of SPI flash, DataFlash, or an SD card) or simulated on a
+block device (one of SPI flash, DataFlash or an SD card) or simulated on a
 heap block device on boards with enough RAM.
 
-By default, this example uses an instance of LittleFS on external SPI flash.
+This example uses an instance of the LittleFileSystem API (LittleFS) on external SPI flash.
```

Comment:
@AnotherButler, why no by default? Do we not want to emphasize out different filesystem/block devices are supported?

Reply:
First, it's unnecessary information making the sentence longer than it needs to be. Second, it's confusing to say an example uses something by default when we mean the example uses something, but you, the user, can change that something.

Reply:
Okeedokee, I can't think of better wording.